recipes/__init__.py

```diff
@@ -16,7 +16,7 @@
 # Since we don't want the tests to to incorrectly assume that recipes are
 # importable, we have to explicitly raise an error here.
 
-# TODO: Add proper link to pytorch.org/torchtune/... when the docs are live.
+# Documentation: (https://pytorch.org/torchtune/stable/index.html)
```

Comment:
let's point to main documentation for now, as that has our latest docs.

Reply:
Of course. All set! Thanks.